alphastats/statistics/DifferentialExpressionAnalysis.py

```diff
@@ -231,17 +244,13 @@ def _pairedttest(self) -> pd.DataFrame:
         df["log2fc"] = fc
         return df
 
-    def _calculate_foldchange(  # TODO duplicated
+    def _calculate_logfoldchange(  # TODO duplicated
         self, mat_transpose: pd.DataFrame, group1_samples: list, group2_samples: list
     ):
         group1_values = mat_transpose[group1_samples].T.mean().values
```

Comment:
please add a docstring making clear that this method assumes log-transformed data

Reply:
Will do in a future pr going over the whole range of docstrings